lexicons/app/bsky/feed/defs.json

```diff
@@ -177,6 +186,81 @@
           "items": { "type": "ref", "ref": "app.bsky.graph.defs#listViewBasic" }
         }
       }
+    },
+    "interaction": {
+      "type": "object",
+      "properties": {
+        "item": { "type": "string", "format": "at-uri" },
+        "event": {
+          "type": "string",
+          "knownValues": [
```

Comment:
If you have any thoughts on `{ item: at-uri, event: token, feedContext: string }` versus open-union-style `{ $type, item: at-uri, feedContext: string, ... }` I'd be interested to hear!

Reply:
Yup that was my first pass on it but then ended up redoing it this way. 

Mainly reconsidered because an array of open unions is tough in languages like Go. But also nearly all these interaction types had similar values on them so they were pretty friendly to squash down into a flat object like this so didn't seem like any resolution was lost